app/js/views/services-list.jsx

```diff
@@ -27,9 +27,17 @@ export default class ServicesList extends React.Component {
   }
 
   isUserFacingService(service) {
+    // If service doesn't have any getters and setters, there is no need to
+    // display it to the user.
+    if (!Object.keys(service.getters).length &&
```

Comment:
It's also short term fix to filter out useless services (like OpenZWave controller), later one I think we'll be requesting getters and setters instead and group them if needed. Maybe.